IFComp/lib/IFComp/Controller/Admin.pm

```diff
@@ -0,0 +1,49 @@
+package IFComp::Controller::Admin;
+use Moose;
+use namespace::autoclean;
+
+BEGIN { extends 'Catalyst::Controller'; }
+
+=head1 NAME
+
+IFComp::Controller::Admin
+
+=head1 DESCRIPTION
+
+A controller for administrative functions
+
+=head1 METHODS
+
+=cut
+
+
+=head2 index
+
+=cut
+
+sub index :Path :Args(0) {
```

Comment:
My version of the schema has role.name as CHAR(8), which means it is not possible for a role to be named 'votecounter'.  Did the schema change?